### DISCUSSION TITLE
UI hosting on different domain name

### DISCUSSION THREAD

**Discussion by jamesgunja** [Netflix/conductor#3020] [General]
How can we point the UI to different path other than root / ? I have tried homepage attribute in packagejson but did not yield the correct result on navigation bar and other resources did not load properly.

**Comment by jamesgunja** [comment]
<BrowserRouter basename=?

**Comment by boney9** [comment]
Hey @jamesgunja  - what set up are you using? Are you building and deploying the docker images? - If yes, then you can modify the nginx conf file to change your route to a different path.

**Reply by jamesgunja** [reply]
Correct - we are building/deploying on docker 

Other than the Nginx config, we had to do also this to reflect.

sed -i 's/"name": "client",/"name": "client","homepage":"custom_root_name",/' package.json
sed -i 's/logo.png/custom_root_name\/logo.png/' src/plugins/AppLogo.jsx
sed -i 's/<BrowserRouter/<BrowserRouter basename="\/custom_root_name"/' src/index.js

**Reply by boney9** [reply]
Awesome - glad to hear you have this working.

**Reply by haricane8133** [reply]
There is one more place where you'd want a change... Links going to a new tab from NavLink by default have a base path of '/'.
I have raised an issue here - https://github.com/Netflix/conductor/issues/3656
I am working on a PR that solves this issue. I will update

**Reply by haricane8133** [reply]
Here is the PR - https://github.com/Netflix/conductor/pull/3722
